package/MDAnalysis/core/universe.py

```diff
@@ -671,8 +672,52 @@ def trajectory(self, value):
         del self._trajectory  # guarantees that files are closed (?)
         self._trajectory = value
 
-    def add_TopologyAttr(self, topologyattr):
-        """Add a new topology attribute."""
+    def add_TopologyAttr(self, topologyattr, values=None):
+        """Add a new topology attribute to the Universe
+
+        Adding a TopologyAttribute to the Universe makes it available to
+        all AtomGroups etc throughout the Universe.
+
+        Parameters
+        ----------
+        topologyattr : TopologyAttr or string
+          Either a MDAnalysis TopologyAttr object or the name of a possible
+          topology attribute.
+        values : np.ndarray, optional
+          If initiating an attribute from a string, the initial values to
+          use.  If not supplied, the new TopologyAttribute will have empty
+          or zero values.
+
+        Example
+        -------
+        For example to add bfactors to a Universe:
+
+        >>> u.add_TopologyAttr('bfactors')
```

Comment:
Stupid question: does `bfactors` exist?

A while back I said https://github.com/MDAnalysis/mdanalysis/issues/1359#issuecomment-333247480 that we renamed it to `tempfactors`. Was I wrong?

Reply:
Yeah we have both as independent Attributes.  These are strictly equivalent right?

So currently MMTF returns "Bfactors" with CRD/PDB/PDBQT returning tempfactors.  So should probably just add an alias to the tempfactors Attribute which allows "`.bfactors`" as well?

Reply:
Yes, they are the same thing.

Reply:
Is `.bfactors` native to MMTF? I'd in principle I like Python's "There is one way of doing it" but then we'd need to decide if what we want to call it – `bfactors` or `tempfactors`. For users it would be easier to access the same TopolAttr with two different names, though. Let's discuss in a separate issue.